airflow/contrib/operators/bigquery_export_query_results_to_gcs.py

```diff
@@ -0,0 +1,138 @@
+from airflow.utils.decorators import apply_defaults
+from airflow.models import BaseOperator
+from airflow.plugins_manager import AirflowPlugin
+from airflow.contrib.hooks.bigquery_hook import BigQueryBaseCursor, BigQueryHook
+from airflow.exceptions import AirflowException
+from uuid import uuid4
+import logging
+
+
+class BigQueryExportQueryResultsToGCS(BaseOperator):
+    """
+    Executes a BigQuery Standard SQL query and exports query results to GCS.
+    For large query results, a GCS * pattern must be present
+    Grant necessary Service Account permission to your output GCS location table
+
+    :param project_id: The GCP project where the query gets executed, temp table is stored and extracted from
+    :type project_id: string
+    :param query: The BigQuery SQL to execute.
+    :type query: string
```

Comment:
Must it be from a string? Can the query be given as a SQL file? Like https://github.com/apache/airflow/blob/master/airflow/gcp/operators/bigquery.py#L367